<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MicrocontrollerWindow</class>
 <widget class="QMainWindow" name="MicrocontrollerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mule Microcontroller Simulator</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="serialLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Microcontroller serial output:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="serialEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="pinTitle">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>151</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Microcontroller pins:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pin0_button">
    <property name="geometry">
     <rect>
      <x>207</x>
      <y>80</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect device</string>
    </property>
   </widget>
   <widget class="QLabel" name="pin0_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>DPIN0</string>
    </property>
   </widget>
   <widget class="QLabel" name="pin1_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>DPIN1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pin1_button">
    <property name="geometry">
     <rect>
      <x>327</x>
      <y>80</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect device</string>
    </property>
   </widget>
   <widget class="QLabel" name="pin2_label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>APIN2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pin2_button">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect sensor</string>
    </property>
   </widget>
   <widget class="QLabel" name="pin3_label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>60</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>APIN3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pin3_button">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect sensor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pin3_pwmbox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>110</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PWM-capable</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="pin4_pwmBox">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>110</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PWM-capable</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="pin3_dcBox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>140</y>
      <width>49</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="pin3_dcLabel">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>140</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Max. DC:</string>
    </property>
   </widget>
   <widget class="QLabel" name="pin4_dcLabel">
    <property name="geometry">
     <rect>
      <x>545</x>
      <y>140</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Max. DC:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="pin4_dcBox">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>140</y>
      <width>49</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="pin0_dval">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Digital value: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="pin1_dval">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>170</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Digital value: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="pin1_dcLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Max. DC:</string>
    </property>
   </widget>
   <widget class="QLabel" name="pin0_dcLabel">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>140</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Max. DC:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pin0_pwmbox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PWM-capable</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="pin0_dcBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>49</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="pin1_dcBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>49</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="pin1_pwmBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PWM-capable</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="pin0_analogdcval">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Current DC: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="pin1_analogdcval">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Current DC: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="pin3_analogdcval">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>200</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Current DC: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="pin4_analogdcval">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>200</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Current DC: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="pin0_cdevstatus">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Nothing is connected</string>
    </property>
   </widget>
   <widget class="QLabel" name="pin1_cdevstatus">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>220</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Nothing is connected</string>
    </property>
   </widget>
   <widget class="QLabel" name="pin3_cdevstatus">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>220</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Nothing is connected</string>
    </property>
   </widget>
   <widget class="QLabel" name="pin4_cdevstatus">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>220</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Nothing is connected</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>678</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Mule_Microcontroller_Simulator"/>
    <addaction name="actionAbout_Qt_framework"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout_Mule_Microcontroller_Simulator">
   <property name="text">
    <string>About Mule Microcontroller Simulator</string>
   </property>
  </action>
  <action name="actionAbout_Qt_framework">
   <property name="text">
    <string>About Qt framework</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
